conf/messages.en

```diff
@@ -218,6 +218,7 @@ smallProducerDetails.remove.hidden = Remove small producer
 
 
 returnChangeRegistration.title = You changed your soft drinks business activity
+returnChangeRegistration.heading = You changed your soft drinks business activity
```

Comment:
Do we really need this duplication? Surely we could use a common key for the two? At least for titles and headings that is.

Reply:
its a common pattern  in mdtp services, as a habit and a pattern I implemented it.